### PR TITLE
feat(gatsby-starter-blog): Post schema tags

### PR DESCRIPTION
**Pull request by marcobiedermann**
## Description

Add schema tags to Post in order to improve SEO

### Documentation

image
image

Reference: https://schema.org/Article

### CODE REVIEW

**Comment by hashimwarren** [comment]
I was going to respond that "Article" isn't one of the schema types Google uses to generate rich results. 

But I just looked it up, and not only does Google now use "Article" but their testing tool came out with support just a few days ago.

So thank you for this timely contribution Marco!

File: starters/blog/src/pages/index.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
+import { graphql, Link } from "gatsby"
```
Comment (LekoArts) [inline starters/blog/src/pages/index.js:1]: Can you revert the changes to the imports please?

**Review by LekoArts — APPROVED** [review]
Thanks for the PR!
